serialization/base.py

```diff
@@ -215,6 +218,20 @@ def _sis_hash(self) -> bytes:
         return sis_hash_helper({"import": super_hash, "hashed_arguments": self.hashed_arguments})
 
 
+class CallImport(PartialImport):
+    """
+    Like PartialImport, but issues a full, not just a partial call to the imported callable.
```

Comment:
The doc is a bit confusing: There is no such thing as a "partial call". There are partial functions.
```suggestion
    Imports some function/class and calls it with given args.

    See also :class:`PartialImport` to get a partial function/class.
```